lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -35,62 +35,29 @@ def setUp(self):
         }
         json_analytics_data = json.dumps(analytics_data)
         self.data = {"data": json_analytics_data}
+        self.zendesk_response = ('A problem has occurred retrieving the data, to report the problem click '
```

Comment:
We should avoid this form of line continuation. Especially since this is already wrapped in parens:

```python
self.zendesk_response = (
    'A problem has occurred retrieving the data, to report the problem click '
    '<a href="https://zendesk.com/hc/en-us/requests/new">here</a>'
)
```

This allows us to remove several levels of superfluous indentation.